lib/tasks/stats.rake

```diff
@@ -19,4 +19,13 @@ namespace :stats do
       DatadogApi.gauge('efile_submissions.state_counts', count, tags: ["current_state:#{state}"])
     end
   end
+
+  desc "Monitor the longest periods of time that any efile submission has been in preparing, bundling, and queued"
+  task monitor_delayed_efile_submissions: :environment do
+    [:preparing, :bundling, :queued].each do |state|
+      oldest_transition_to = EfileSubmissionTransition.where(to_state: state).sort_by(&:created_at).first
```

Comment:
I think you want to scope this to EfileSubmissionTransition where most_recent is true, or else this is going to give you the same oldest transition on every run.

Reply:
ohhh good catch!